### PR TITLE
LOOP-3904 Change 2s threshold to 6s

### PR DESCRIPTION
**Pull request by ps2**
https://tidepool.atlassian.net/browse/LOOP-3904

### CODE REVIEW

File: DashKit/DashPumpManager.swift [hunk 1]
```diff
@@ -37,6 +37,8 @@ open class DashPumpManager: PumpManager {
         return Pod.maximumBasalScheduleEntryCount
     }
     
+    private static let MaxTempBasalGapFillDuration = TimeInterval(seconds: 6)
```
Comment (rickpasetto) [inline DashKit/DashPumpManager.swift]: [nit] I don't think we capitalize variables, even constants, anywhere else.  Consider `maxTempBasalGapFillDuration`

**Review by rickpasetto — APPROVED** [review]
One nit, other than that LGTM

**Review by darinkrauss — APPROVED** [review]
LGTM!

File: DashKit/DashPumpManager.swift (hunk 1 above)
Comment (ps2) [reply DashKit/DashPumpManager.swift]: Haha, yeah, slipping back into objective c naming. There are a few other uppercase first letter constants still lingering in our swift code, looks like, in Loop; mainly for notification names.

**Pull request merged**